packages/design-system/src/components/IdleTimeout/IdleTimeout.tsx

```diff
@@ -85,6 +85,10 @@ const defaultMessageFormatter = (timeTilTimeout: number): React.ReactNode => {
 // local storage variable name
 const lastActiveCookieName = 'CMS_DS_IT_LAST_ACTIVE';
 
+/**
+ * For information about how and when to use this component,
+ * [refer to its full documentation page](https://design.cms.gov/components/idle-timeout/).
+ */
```

Comment:
This is probably something for another Jira ticket, but the story for Idle Timeout is a little goofy. Because the modal doesn't have open/close functionality, it's always open and blocks the props table.

Reply:
I don't understand this one. On my computer, the story is disabled on the doc page

Reply:
I don't see it popping up

Reply:
Sorry... it was the privacy modal not the idle modal

Reply:
Okay fixed!